devito/passes/clusters/aliases.py

```diff
@@ -80,22 +65,32 @@ def cire(cluster, template, mode, options, platform):
     t0 = 2.0*t2[x,y,z]
     t1 = 3.0*t2[x,y,z+1]
     """
-    # Sanity checks
-    assert mode in ['invariants', 'sops']
-    assert all(i > 0 for i in options['cire-repeats'].values())
-
     # Relevant options
     min_storage = options['min-storage']
+    min_cost = options['cire-mincost']
+    repeats = options['cire-repeats']
+
+    # Sanity checks
+    assert mode in ['invariants', 'sops']
+    assert all(i > 0 for i in repeats.values())
```

Comment:
do we want to allow `0` to get a zero optization operator? just a thought

Reply:
I could allow it... but if one really wants to disable it, the way to go is to use a CustomOperator as we do in some tests btw

Reply:
I get that but this would be an "easier" way to have quick access to a basic noop operator.

Reply:
OK, fine by me. Do you mind if I do that in the PR that sits right on top of this? it's the patch for the bug we discovered yesterday. So we don't have to respin CI, rebase, etc

Reply:
Completely fine yes. Played with that other one and looks like it fixes the issue so can probably open the PR (that big try/except not sure but does the job).